### ISSUE TITLE
New ticket button on TECH's company page.

### ISSUE THREAD

**Issue by Nness** [DeskDirector/UserVoice#563]
IT would be helpful to create a ticket in Tech from the Company page
I will normally look to see if the ticket exists already so this is where I normally start a ticket in tech for a client 

![image](https://user-images.githubusercontent.com/7351746/120396108-221a8a80-c38a-11eb-9103-a928634a226b.png)

DeskDirector/Issues#220

**Comment by Chrispe66** [comment]
Yes, I second this. Maybe just move the New button to the top level so we can, at any time, create a new ticket. However, if you are going to the Company page to look at tickets, why not move to the ticket page and filter by a company?
